ppr-ui/src/components/unitNotes/UnitNoteAdd.vue

```diff
@@ -136,7 +136,7 @@ export default defineComponent({
           : personGivingNoticeContent
       ),
       isNoticeOfTaxSale: computed((): boolean => props.docType === UnitNoteDocTypes.NOTICE_OF_TAX_SALE),
-      hasNoPersonGivingNotice: (getMhrUnitNote.value as UnitNoteIF).hasNoPersonGivingNotice || false,
+      hasNoPersonGivingNotice: props.docType === UnitNoteDocTypes.DECAL_REPLACEMENT || (getMhrUnitNote.value as UnitNoteIF).hasNoPersonGivingNotice || false,
```

Comment:
Looks good to me!

Also want to present an option: You could just watch the `props.docType` and if/when it's `UnitNoteDocTypes.DECAL_REPLACEMENT` set the localState/State and the validation in the same function.